src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataToRead = getDataFromFile(fromFileName);
+        String dataToWrite = dataProcessing(dataToRead);
+        writeDataToFile(toFileName, dataToWrite);
+    }
+
+    private String[] getDataFromFile(String fileName) {
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
```

Comment:
Please use try-with-resources as described in common mistakes. Also, you can pass just a filename to the FileReader constructor instead of File instance